records_mover/records/targets/data_url.py

```diff
@@ -16,7 +14,6 @@
 
 
 class DataUrlTarget(SupportsMoveFromDataframes,
```

Comment:
Targets in Records Mover represent the second part in a move (e.g., in a Redshift to S3 copy, S3 is the target and Redshift is the source).

DataUrlTarget represents any kind of filesystem-like thing - currently it works with `file://`, `gs://` and `s3://` URLs, but will probably get `sftp://` support at some point as well.